utils/transformers.py

```diff
@@ -16,9 +16,10 @@
 def load_embedding_vectors(embedding_style, file=None):
     """
     Helper function in order to load word2vec or glove vector embedding files provided on https://deepset.ai/german-word-embeddings.
+
     Args: 
       embedding_style: {'word2vec', 'glove'}, default=None
-      file: str Path of the embeddingFile. default is None.
+      file: An embedding vector file containing words and their corresponding embedding vectors.
```

Comment:
You could change it to "An embedding vector file containing words and their corresponding embedding vectors (as float 32)."
Now, making this suggestiong, isn't the reading of the file very specific to the files, that we downloaded? Maybe it makes sense to add the link to the page in the docstring?

Reply:
the link is already there In line 18 :-)

Reply:
Ups. Obviously I did't see it.